graspologic/embed/n2v.py

```diff
@@ -128,7 +130,13 @@ def node2vec_embed(
         f"Completed. Ending time is {str(end)} Elapsed time is {str(start - end)}"
     )
 
-    return model.wv.vectors, model.wv.index2word
+    labels = node2vec_graph.original_graph.nodes()
+    remapped_labels = node2vec_graph.label_map_to_string
+
+    return (
+        np.array([model.wv.get_vector(remapped_labels[node]) for node in labels]),
+        np.array(labels),
```

Comment:
should we np.array this and not just [] it? the reason I bring it up is technically networkx is going to be fine with you mixing the node types (ints and strings), whereas I'm not sure what np.array does if you send it a list of different types?

Then again .nodes() may be returning some sort of AtlasView object or something and numpy may be fine saying the type is of AltasView or something?

Reply:
Modified to return lists instead. No real reason to return np.arrays, was just trying to maintain the current api.